tests/common.go

```diff
@@ -2085,7 +2086,7 @@ func ScheduleValidateClusterPair(ctx *scheduler.Context, skipStorage, resetConfi
 		}
 	}
 
-	pairInfo, err := Inst().V.GetClusterPairingInfo(kubeConfigPath, "")
+	pairInfo, err := Inst().V.GetClusterPairingInfo(kubeConfigPath, "", IsEks())
```

Comment:
This `IsEks()` function only works with `StorageCluster`, I see we are using `ScheduleValidateClusterPair()` in https://github.com/portworx/torpedo/blob/01bcf306d661a6de380e591442f9d1fa32df9236/tests/basic/async_dr_test.go#L85 but I don't see that we are verifying that this can only run on operator based deployments. What if this test is running on ds deployment?

Reply:
Makes sense the name of the method is misleading, changed name to `IsEksPxOperator` to reflect what the function is doing.